src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -64,7 +64,12 @@ public class OtlpExporterOptions
         /// <summary>
         /// Gets or sets optional headers for the connection.
         /// </summary>
-        public Metadata Headers { get; set; } = new Metadata();
+        public string Headers { get; set; }
```

Comment:
We need to document the format here. Or do something like IEnumerable<KeyValuePair<string, string>> which is a little more obvious to users how it works (hopefully).

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specifying-headers-via-environment-variables
We can keep it string, and link to the spec.